src/Database/Attach/File.php

```diff
@@ -105,7 +108,10 @@ public function fromFile($filePath)
         $this->file_size = $file->getSize();
         $this->content_type = $file->getMimeType();
         $this->disk_name = $this->getDiskName();
-
+        $realpath = ( empty(trim($file->getRealPath())) 
```

Comment:
Without these checks/backup fillings you get these kind of messages. Unfortunately no traces in the logs or the php error log of this.

![capture](https://cloud.githubusercontent.com/assets/8638425/13993798/2df8de6e-f122-11e5-91aa-b53f802c287c.JPG)